atoma-state/src/state_manager.rs

```diff
@@ -987,23 +988,26 @@ impl AtomaState {
                 key_rotation_counter, 
                 node_small_id, 
                 public_key_bytes, 
-                remote_attestation_bytes, 
+                remote_attestation_bytes,
+                certificate_chain_bytes,
                 device_type
             ) 
-            VALUES ($1, $2, $3, $4, $5, $6)
+            VALUES ($1, $2, $3, $4, $5, $6, $7)
             ON CONFLICT (node_small_id, device_type)
             DO UPDATE SET
                 epoch = $1,
                 key_rotation_counter = $2,
                 public_key_bytes = $4,
-                remote_attestation_bytes = $5
+                remote_attestation_bytes = $5,
```

Comment:
btw instead of $5 on conflict you can do stuff like : `remote_attestation_bytes = EXCLUDED.remote_attestation_bytes` 
you don't have to change it, I just like it better, because it's easier to see it's the correct value

Reply:
Will change